### PR TITLE
Prevent too many xeno at start

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Xeno get two "free" jobs at roundstart. This is bad because it kinda make the ratio xeno/marines not linear with pop, so this prevents it. The free jobs are needed to let the job system works

Also give back the ratio roundstart xeno vs marine i made 3 months ago, cause apparently marines cannot handle old ratio anymore

Make each silo ramp up the output production more (from 33% to 40%), to compensate to low xeno starting points

Changes crash initial xeno/marines ratio

## Why It's Good For The Game

Consistent ratio.

Also fixes crash ratio that was completly boinkers the only round we played (It was even 12 previously)

## Changelog
:cl:
balance: Remove 2 xenos jobs at roundstart, no matter the pop
balance: Less xeno in crash
balance: Less xeno for marines overall
/:cl:

### CODE REVIEW

**Comment by David-Stormwell** [comment]
Unneeded from what I've personally seen, if anything make it simply -1?

**Comment by SplinterGP** [comment]
Crash has a problem(actually every gamemode has it) of low amount of xenis being way too weak while any amount over that is way way stronger, so at crash id like something like minimum amount of xenos

**Comment by SplinterGP** [comment]
Ive noted this is not only a problem at crash since some days ago we were with a small pop like 50 to 60, xeno numbers wouondt be that big and that caused them to lose frequently (70% of time)

**Comment by EgorDinamit** [comment]
Now prevent rounds from having too many marines.

**Comment by KornFlaks** [comment]
11 xenos vs 42. So good.  Xenos been winning too much anyways. I say speedmerge.

**Comment by Al-1ce** [comment]
Xeno pop isn't linear in accordance to marine pop anyways, since for every Xeno there are generally 5-10 marines.
This feels completely unnecessary and will just end up being a nerf in the long run, those two extra Xenos (commonly drones) tend to make the difference between the map being weeded and not, they may also aid in stopping a full steamroll should they opt for combat castes.

33% - 40% will not matter when you don't have enough ayys to actually defend the Silo, or that the ones you have are barely fresh larva & young.


> Also give back the ratio roundstart xeno vs marine i made 3 months ago, cause apparently marines cannot handle old ratio anymore


Name five people who cannot handle the old ratio.

**Comment by SplinterGP** [comment]
> Name five people who cannot handle the old ratio.
the old ratio was not ingame has some lots of time

**Comment by Al-1ce** [comment]
>the old ratio was not ingame has some lots of time

Has..some lots of time? I'm not sure what you intended to say at the end there, but then nothing changes if it has never been implemented. I'll say it again, this TM is unnecessary.

**Comment by BraveMole** [comment]
> > the old ratio was not ingame has some lots of time
> 
> Has..some lots of time? I'm not sure what you intended to say at the end there, but then nothing changes if it has never been implemented. I'll say it again, this TM is unnecessary.

This was the ratio for 4 months ace. I just modified it 1 week ago, but almost all of pools was played with it

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by Passe2142** [comment]
Mm the thing was that on crash some funny xenos stored dead colonists and memerines. Crash wise this is chill.

File: code/datums/jobs/job/xenomorph.dm [hunk 1]
```diff
@@ -14,6 +14,11 @@
 		<b>Gamemode Availability</b>: Crash, Distress<br /><br /><br />
 		<b>Duty</b>: Spread the hive, obey the will of your Hive Leader and the Queen Mother. Kill or capture those who get into your way. Protect the hive whenever possible. Amass your numbers.
 	"}
+	/**
+	 * This is the amount of "free" xeno jobs opened at the start, even before marines job contributes to the total.
+	 * This is a counter to prevent adding more jobs that necessary
+	 */
+	var/free_xeno_at_start = FREE_XENO_AT_START
```
[the remark this answers is not in the document]
Comment (BraveMole) [reply code/datums/jobs/job/xenomorph.dm:21]: both the define and the var is used?

**Review by TiviPlus — APPROVED** [review]
Gonna assume since noones saying anything this is fine

**Pull request merged**